### PR TITLE
Get & post apis for passwords

### PR DESCRIPTION
**Pull request by CodeWUsama**

### CODE REVIEW

File: controllers/passwords.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from '@prisma/client';
+import sendResponse from '../helpers/responseHelper.js';
+import createPasswordSchema from '../validationSchemas/passwordSchema.js';
+
+const prisma = new PrismaClient();
+
+export const handleGetPasswords = async (req, res) => {
```
Comment (beinghaziq) [inline controllers/passwords.js:7]: what does this method do?

Comment (usamabasharat) [reply controllers/passwords.js:7]: Fetch all the passwords from DB
Route: '/passwords'
Method: GET
We might alter it later. I made this to see if passwords are saving into DB or not.

Comment (beinghaziq) [reply controllers/passwords.js:7]: We have to remove this method later. Because we might not want to fetch passwords this way

Comment (beinghaziq) [reply controllers/passwords.js:7]: This can be a security flaw

Comment (CodeWUsama) [reply controllers/passwords.js:7]: Yes, this is definitely gonna be removed. It is just used for testing purposes.

**Pull request merged**